evergreen.yml

```
@@ -2043,6 +2047,7 @@ tasks:
       #   variants: [ ubuntu2204 ]
 
   - name: sign
+    allowed_requesters: ["patch", "github_tag"]
```

```suggestion
    allowed_requesters: ["adhoc", "github_tag"]
```

Patch is for when you run a patch to test something and I think for PR checks.
adhoc is for periodic builds.

---

ah yes, thanks. I got turned around by the wording in the ticket